app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt

```diff
@@ -245,9 +245,9 @@ private fun GovUkNavHost(
         homeGraph(
             widgets = homeScreenWidgets(
                 navController = navController,
-                isSearchEnabled = uiState.isSearchEnabled,
-                isTopicsEnabled = uiState.isTopicsEnabled,
-                isRecentActivityEnabled = uiState.isRecentActivityEnabled,
+                isSearchEnabled = if (uiState is AppUiState.Default) uiState.isSearchEnabled else false,
```

Comment:
You can remove these checks if you change the ui state argument type to AppUiState.Default

Reply:
Ah yes! Thanks, done.